### ISSUE TITLE
NTR: N-ribosylhistidine high in urine (Lab: urine tests)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8850]
New term request N-ribosylhistidine high in urine (elthat_111113035644):

**Comment by pnrobinson** [comment]
@MickeySegal
https://pubmed.ncbi.nlm.nih.gov/4030756/
is a tau missing?

**Comment by MickeySegal** [comment]
My OMMBID source seems to have dropped the ball here too.  I changed our finding to "N τ-ribosylhistidine high in urine"

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000621
- label: Elevated urinary N-tau-ribosylhistidine level
- parents: [Increased urine proteinogenic amino acid derivative level (HP:0033097)](https://hpo.jax.org/app/browse/term/HP:0033097)
- pmid: PMID:2992472
- def: Presence of N tau-ribosylhistidine (His-R) in the urine. His-R is a histidine derivative found in the urine of histidinemic patients.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8850
- synonyms: Elevated urinary N-ribosylhistidine level(NONE)
